x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_item.tsx

```diff
@@ -7,44 +7,43 @@
 import React, { useState } from 'react';
 import DateMath from '@elastic/datemath';
 import {
+  EuiButtonGroup,
   EuiFlexGroup,
   EuiFlexItem,
-  EuiProgress,
-  EuiPopover,
-  EuiLoadingSpinner,
+  EuiIconTip,
   EuiKeyboardAccessible,
-  EuiText,
-  EuiToolTip,
-  EuiButtonGroup,
+  EuiLoadingSpinner,
+  EuiPopover,
   EuiPopoverFooter,
   EuiPopoverTitle,
-  EuiIconTip,
+  EuiProgress,
```

Comment:
Not sure that this changes related to your PR

Reply:
Prettier did alphabetical order.